.github/workflows/deploy.yml

```diff
@@ -0,0 +1,34 @@
+name: Continuous Deployment
+on:
+  push:
+    branches:
+      - master
+      - develop
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v1
+
+        - name: Install dependencies
+          run: yarn
```

Comment:
Why using yarn ? A JS package manager for Python Project !

Reply:
So i need to remove yarn as it is not useful. What else do i need to add or remove?

Reply:
@keshakaneria Have you used travis CI ?